chart/epinio/questions.yml

```diff
@@ -0,0 +1,166 @@
+  label: Email
+  description: "Email to use for getting notifications about your certificates"
+  default: "epinio@suse.com"
+  type: strings
+  required: false
+  group: "General settings"
+- variable: certManagerNamespace
+  label: Cert-manager namespace
+  description: "Namespace where cert-manager is installed in"
+  default: "cert-manager"
+  type: strings
+  required: false
+  group: "Advanced settings"
+- variable: ingress.ingressClassName
+  label: Ingress class name for the server
+  description: "Set a class name to select the ingress controller you want to use for the server"
+  default: ""
+  type: strings
+  group: "Advanced settings"
+- variable: server.ingressClassName
+  label: Ingress class name for apps
+  description: "Set a class name to select the ingress controller you want to use for your apps"
+  default: ""
+  type: strings
+  group: "Advanced settings"
+- variable: useCustomTlsIssuer
+  label: Use your own TLS issuer
+  default: "false"
+  description: "Use your own TLS issuer"
+  type: boolean
+  group: "General settings"
+  show_subquestion_if: true
+  subquestions:
+  - variable: customTlsIssuer
+    label: TLS issuer
+    description: "Name of the cluster issuer to use"
+    type: strings
+    required: false
+- variable: global.tlsIssuer
+  show_if: "useCustomTlsIssuer=false"
+  label: TLS issuer
+  description: "Name of the predefined cluster issuer to use"
+  default: "epinio-ca"
+  type: enum
+  required: false
+  group: "General settings"
+  options:
+    - "epinio-ca"
+    - "selfsigned-issuer"
+    - "letsencrypt-production"
+- variable: api.username
+  label: API username
+  description: "The user name for authenticating all API requests"
+  type: strings
+  required: false
+  group: "General settings"
+- variable: api.password
+  label: API password
+  description: "The password for authenticating all API requests"
+  type: password
+  required: false
+  group: "General settings"
+- variable: global.domain
+  label: Domain
+  description: "Domain for the application"
+  type: strings
+  required: true
+  group: "General settings"
+- variable: server.accessControlAllowOrigin
+  label: Access control allow origin
+  description: "Domain which serves the Rancher UI (to access the API)"
+  type: strings
+  required: false
+  group: "General settings"
+- variable: kubed.enabled
+  label: Install kubed
+  description: "Deploy kubed or skip it if you get it installed already"
+  type: boolean
+  default: true
+  group: "Advanced settings"
+- variable: containerregistry.enabled
+  default: "true"
+  description: "Disable local container registry to configure an external registry."
+  label: Install local container registry
+  type: boolean
+  show_subquestion_if: false
+  group: "External registry"
+  subquestions:
+  - variable: registry.url
+    label: External registry url
+    description: "URL of your external registry"
+    type: strings
+    required: false
+  - variable: registry.username
```

Comment:
shouldn't this be `global.registryUsername` ? same for the others in this section

Reply:
oh yes good point, it's ok on my dev env and I forgot to change them here...